### DISCUSSION TITLE
Changing screen resolution when unplugged

### DISCUSSION THREAD

**Discussion by keeoooni** [seerge/g-helper#1293] [Ideas]
I think it would be pretty nice for battery life to have the option to automatically lower screen resolution when on battery. I use 1440p when plugged in, but would be cool if I didn't have to go all the way through my device settings to lower my screen resolution to 1080p.

**Comment by seerge** [comment]
@keeoooni hello, what is the practical sense of changing resolution in windows? (it makes things look blurry and doesn't affect battery life anyhow)

**Reply by Scarlaid** [reply]
oddly sounds like they have mistaken resolution to refresh rate, which is actually what affect battery life 🗿

**Reply by IceStormNG** [reply]
> oddly sounds like they have mistaken resolution to refresh rate, which is actually what affect battery life 🗿

I think they really mean resolution. There are posts of people around the internet that lower the screen res to save battery and recommend that as a great battery saving tip, not realizing that the gains are so low that you just ruin your image quality on the screen when you don't drive the LCD at its native resolution.
